ocaml/xapi/xapi_session.ml

```diff
@@ -742,6 +758,10 @@ let slave_local_login_with_password ~__context ~uname ~pwd =
       against the local superuser credentials
 *)
 let login_with_password ~__context ~uname ~pwd ~version:_ ~originator =
+  Context.with_tracing
+    ~attributes:[("xs.xapi.session.originator", originator)]
```

Comment:
this is repeated above, we need a single place that creates this attribute, perhaps a function attributes_of_originator in session.ml: Session.Tracing.attributes_of_originator?

Reply:
@psafont  @mg12 Something like this? https://github.com/xapi-project/xen-api/pull/5621/commits/1aedd6ddb9867debe7cbc3e5ae9e3e5c9ba1d17d